doc/howto.md

```diff
@@ -11,6 +11,19 @@ kernelspec:
   name: python3
 ---
 
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+# clean up all .db files (this cell will not be displayed in the docs)
```

Comment:
According to the comment this cell shouldn't be displayed in the docs but [hide-cell] hides the cell and the user can reveal it.

We can remove this cell with [remove-cell]. [remove an entire code cell](https://jupyterbook.org/en/stable/interactive/hiding.html#remove-an-entire-code-cell)